include/votca/tools/elements.h

```diff
@@ -84,7 +84,7 @@ class Elements {
   /// Returns the mass of each atom in a.u.
   const double &getMass(std::string name) const { return _Mass.at(name); }
   /// Returns the covalent Radii of the atom 
-  const double getCovRad(std::string name,std::string unit ) const 
+  double getCovRad(std::string name,std::string unit ) const 
```

Comment:
Why?

Reply:
gives compiler warnings, because you return a variable by copy but const declare it